NumberBaseball/NumberBaseball/main.swift

```diff
@@ -76,13 +142,27 @@ func checkStrike(_ strike: Int, _ strikeCount: Int = 3) -> Bool {
     }
 }
 
-func addScore(to resultStrikeAndBall: inout [Int], computer: Array<Int>.Index, user: Array<Int>.Index) {
+func calculateResult(computerNumbers: [Int], userNumbers: [Int]) -> [Int] {
+    
+    var resultStrikeAndBall: [Int] = [0, 0]
+    
+    for userIndex in 0...(userNumbers.count - 1) {
```

Comment:
```suggestion
    for userIndex in userNumbers.indices {
```
해당 CollectionType의 유효한 인덱스의 범위를 리턴해주는 [indices](https://developer.apple.com/documentation/swift/collection/1641719-indices) 프로퍼티를 사용하면 좀 더 깔끔한 코드가 될 것 같네용👍

Reply:
indices라는 프로퍼티가 있는 줄 처음 알았네요!
앞으로도 정말 유용하게 사용할 것 같습니다..!
감사합니다 🙏

Reply:
저도 처음 알게 되었네요! 배열에서 유용하게 사용할 것 같아요! 감사합니다 마이노 !!